plugins/execute/transformerExporter.go

```diff
@@ -146,140 +148,141 @@ var Exporter exporter
 
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
 	return []event.TransformerInitializer{
-			auction_file.EventTransformerInitializer,
```

Comment:
This diff is a bit challenging to read, not sure why things were re-sorted so that they're not in alphabetical order. Not a big deal, but just want to verify that the only change in here is that the collateral generator added `flip_univ2ethusdt_a_v1_2_5.StorageTransformerInitializer` and `flip_univ2daiusdc_a_v1_2_5.StorageTransformerInitializer` to this the `storage.TransformerInitializer` slice. The `event.TransformerInitializer` slice shouldn't have changed.

Reply:
👍 yes the changes are to the `storage.TransformerInitializer ` slice. Good idea on adding a test on the  `FlipV110ABI()` method. That's super easy to miss.